sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcIOTest.java

```diff
@@ -108,14 +112,14 @@ public static void startDatabase() throws Exception {
     dataSource.setServerName("localhost");
     dataSource.setPortNumber(port);
 
-
-    JdbcTestDataSet.createReadDataTable(dataSource);
+    JdbcTestHelper.createDataTable(dataSource, JdbcTestHelper.READ_TABLE_NAME);
```

Comment:
I switched it so that the table names are all generated via getTableName - check it out.

I also did the coordination across tests where it makes sense - the read tests share a single table since they all want to read from the same source (that table will still be unique to the particular test run.)